webapp-client/src/component/Item.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+
+class Item extends Component {
+  formatSourceLink = () => {
```

Comment:
it is actually recommended to use bound functions rather than arrow functions: https://reactjs.org/docs/handling-events.html

Reply:
That actually shouldn't be in this PR! Oops! It's fixed in my draft PR #30